package.json

```diff
@@ -93,6 +97,9 @@
     "yargs": "3.32.0",
     "yargs-parser": "7.0.0"
   },
+  "resolutions": {
```

Comment:
had to add this as `tsc` needs one version of node's `d.ts` file, and `chrome-launcher` has a specific version while everyone else just does `*`. See [this discussion](https://github.com/yarnpkg/yarn/issues/3951#issuecomment-327864441) (and links in that thread) for background